.github/workflows/wheel-builder.yml

```diff
@@ -271,6 +298,9 @@ jobs:
           name: cryptography-sdist
 
       - run: ${{ matrix.PYTHON.BIN_PATH }} -m pip install -r "${UV_REQUIREMENTS_PATH}"
+      - name: add free-threaded python tools directory to PATH
+        run: echo "/Library/Frameworks/PythonT.framework/Versions/3.14/bin" >> "$GITHUB_PATH"
+        if: matrix.PYTHON.VERSION == '3.14t'
```

Comment:
Ooof, why does this need to be special cased?

Reply:
Oh actually on second thought I might be able to delete this, let me see if I can remember why I added this, I don't think my last comment is the correct reason...

Reply:
OK, now I remember. This is described in the mac installer notes that I linked above, on the last bullet point under "known limitations". The GIL-enabled build adds the search path to PATH by default as part of the installer process, but doesn't add the `PythonT.framework` path, so we have to add it manually.


I think @ned-deily is planning to update the installer eventually to fix these papercuts but those updates didn't make it in time for 3.14.

Reply:
And I can't use `python -m uv`, because that doesn't behave the same way as plain `uv` does.